plugin/mutate/source/dextool/plugin/mutate/backend/test_mutant/package.d

```diff
@@ -1118,7 +1118,17 @@ nothrow:
         auto files = spinSql!(() => db.getFiles());
         const fileScores = reportScores(*db, kinds, files);
         const score = reportScore(*db, kinds);
-        const time = Clock.currTime;
+        auto time = Clock.currTime.toUTC;
+
+        try {
+            time.hour = 0;
```

Comment:
i think this is better to move to putMutationScore because then we "know" that it is always zeroed just before it is saved to the DB wherever it is called in the future.